src/plugins/kernel_install/raspbian.sh

```diff
@@ -0,0 +1,33 @@
+# Kworkflow treats this script as a plugin for installing a new Kernel or
+# module in a target system. It is essential to highlight that this file
+# follows an API that can be seen in the "deploy.sh" file, if you make any
+# changes here, you must adjust install_modules() and install_kernel()
+# accordingly.
+#
+
+# Update boot loader API
+function update_raspbian_boot_loader()
+{
+  local name="$1"
+  local target="$2"
+  local flag="$3"
+
+  # https://www.raspberrypi.org/documentation/configuration/config-txt/boot.md
+
+  if grep --quiet "kernel" /boot/config.txt; then
+    sudo sed -ri "/kernel/s/=.+/=$name/g" /boot/config.txt
+  else
+    echo "kernel=$name" | sudo tee -a /boot/config.txt
+  fi
+
+}
+
+function generate_raspbian_temporary_root_file_system()
+{
+  local name="$1"
+  local target="$2"
+  local flag="$3"
+
+  # there is nothing do be done in th rpi case
```

Comment:
não entendi

Reply:
kw is structured in a way that every plugin/ distribution needs to have at least 2 functions, one for the bootloader and one for the temporary root file system like initramfs. But afaik there is nothing equivalent to the initramfs in the raspberry. Since the function needs to exist, I just created a dummy function that returns immediately

Reply:
Yeah, maybe we can improve this in the future but for now, add a comment in the function doc.